components/Header.tsx

```diff
@@ -1,74 +1,130 @@
 // Copyright 2023 the Deno authors. All rights reserved. MIT license.
 import {
   ACTIVE_LINK_STYLES,
-  BUTTON_STYLES,
   LINK_STYLES,
-  NAV_STYLES,
   SITE_BAR_STYLES,
   SITE_NAME,
 } from "@/utils/constants.ts";
 import { stripe } from "@/utils/payments.ts";
-import { Bell, CircleFilled } from "./Icons.tsx";
-import { getToggledStyles } from "@/utils/display.ts";
+import { Bars, Bell, CircleFilled, Cross } from "./Icons.tsx";
+import { cx } from "@twind/core";
+import type { JSX } from "preact";
+
+interface MenuItemProps {
+  name: string | JSX.Element;
+  href: string;
+  active?: boolean;
+  class?: string;
+  ariaLabel?: string;
+}
+
+function MenuItem(props: MenuItemProps) {
+  return (
+    <a
+      href={props.href}
+      class={cx(
+        props.active ? ACTIVE_LINK_STYLES : LINK_STYLES,
+        "relative text-gray-500 px-3 py-4 sm:py-2",
+        props.class,
+      )}
+      aria-label={props.ariaLabel}
+    >
+      {props.name}
+    </a>
+  );
+}
 
 export default function Header(
   props: { sessionId?: string; hasNotifications: boolean; url: URL },
 ) {
+  const menuItems: (MenuItemProps | undefined)[] = [
+    stripe && { name: "Pricing", href: "/pricing" },
+    props.sessionId
+      ? { name: "Account", href: "/account" }
+      : { name: "Sign in", href: "/signin" },
+    {
+      name: (
+        <>
+          <div class="relative">
+            <Bell class="hidden sm:block w-6 h-6" />
+            <span class="sm:hidden">
+              Notifications
+            </span>
+            {props.hasNotifications && (
+              <CircleFilled class="absolute -top-0.5 -right-0.5 text-primary w-2 h-2" />
+            )}
+          </div>
+        </>
+      ),
+      href: "/account/notifications",
+      ariaLabel: "Notifications",
+    },
+    { name: "Submit a project", href: "/submit", class: "sm:hidden" },
```

Comment:
```suggestion
    { name: "Submit", href: "/submit", class: "sm:hidden" },
```
I'm happy to keep the inner text the same.

Reply:
Isn't it a little vague what to do if the label is just "Submit"? In PH, the label is only Submit, but the target is selected by hover.

<img width="429" alt="image" src="https://github.com/denoland/saaskit/assets/3132889/35026be1-8dc7-4b8c-a735-5a48385f85bf">

Rather, what if the header button to say "Submit a project" when it is wide enough?